modules/content_filter/tests/test_normalizer.py

```
@@ -0,0 +1,68 @@
+HANGUL_FILLER = chr(0x3164)
+
+
+@pytest.mark.parametrize(
+    "raw, expected_despaced",
+    [
+        ("노 래 방", "노래방"),  # 공백 삽입
+        ("노.래.방", "노래방"),  # 구분 기호 삽입
+        ("노^^래방", "노래방"),  # 특수문자 삽입
+        (f"노{ZERO_WIDTH}래{ZERO_WIDTH}방", "노래방"),  # zero-width
+        (f"노{HANGUL_FILLER}래방", "노래방"),  # 한글 필러
+        ("ＮＦＫＣ", "NFKC"),  # 전각 → 반각 (NFKC)
```

Ruff (0.15.17)

[warning] String contains ambiguous `Ｎ` (FULLWIDTH LATIN CAPITAL LETTER N). Did you mean `N` (LATIN CAPITAL LETTER N)?

(RUF001)